internal/fflag/fflag.go

```diff
@@ -0,0 +1,133 @@
+package fflag
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/launchdarkly/go-sdk-common/v3/ldcontext"
+	"github.com/superfly/flyctl/internal/tracing"
+)
+
+const clientSideID string = "6557a71bbffb5f134b84b15c"
+
+type FeatureFlagClient struct {
+	ldContext  ldcontext.Context
+	flags      map[string]FeatureFlag
+	flagsMutex sync.Mutex
+}
```

Comment:
You can just call this `Client` since "feature flag" is implied from the package name.